.github/workflows/en-updates-to-translation-repo.yml

```diff
@@ -0,0 +1,44 @@
+name: Push EN changes to translations branch
+
+on:
+  push:
+    # Only run when EN _i18n files are touched - see https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
+    paths:
+      - _i18n/en.yml
+      - _i18n/en/**
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  update-files:
+    name: Update PO files
+    runs-on: ubuntu-latest
+    steps:
+        # checks out your repository and downloads it to the runner, allowing you to run actions against your code
+      - name: Get repository
+        uses: actions/checkout@v2
+      - name: Install mdpo
+        run: pip install mdpo
+      - name: Create folder for files to be copied
+        run: mkdir _i18n/translation-files
+      - name: Create/update YAML file with short texts
+        # translated files are not copied because translation is done in Weblate - any manual translation edits will be lost
+        run: cp _i18n/en.yml _i18n/translation-files
+      - name: Create/update POT file of Contribute section
```

Comment:
All project pages (not the YAML file with short strings) could be merged indeed. Did you mean to merge everything, or just put the contribute and general pages together?
The upside of keeping contribute and general pages separate is that we could give general pages higher priority. It would also be possible I think to later add an automatic check to require 100% completion for general & documentation pages (but not contribution), before a website language is added. No strong feelings, however.

Reply:
> Did you mean to merge everything, or just put the contribute and general pages together?

Just contribute+general (except yaml, for technical reasons)

> It would also be possible I think to later add an automatic check to require 100% completion for general & documentation pages (but not contribution)

I would actually do 100% general+contribute but not documentation :D The documentation probably changes more often than general+contribute

Reply:
Hmm. Do you mind if we keep it separate, then? It doesn't really matter from a technical perspective and there's not a massive difference for translators, but then we can decide later on the percentages :)

Reply:
Okay, let's keep them separate